### PR TITLE
Contributors logo

### PR DESCRIPTION
**Pull request by vrunda2005**
## Related Issue  
Fixes **#26: [CI] Contributors logo in README.md doesn't update in real-time**  

## Description  
- Added a new workflow file: `workflows/contributors_update.yml`.  
- Updated the `README.md` section where the contributors' logos are displayed to ensure it updates in real-time.  

---

## Checklist  
Please ensure the following before submitting the PR:  
- [x] I have read and followed the guidelines in the `CODE_OF_CONDUCT.md` file.  
- [ ] I have followed the design specified in the Figma file (if applicable).  

---

## Additional Comments  
- Through this PR, I learned about GitHub Actions.  
- However, I am still unsure how to verify if the workflow is functioning correctly. Any guidance on testing it would be appreciated.

### CODE REVIEW

**Comment by Saumya40-codes** [comment]
> * However, I am still unsure how to verify if the workflow is functioning correctly. Any guidance on testing it would be appreciated.

You can create a private repository and add the same workflow in `.github/workflows/` directory there, then can try out by pushing (or any other action need to trigger that workflow)

**Comment by vrunda2005** [comment]
Does this work?In my private repo this code is working.
Screenshot 2024-12-26 210651

**Comment by Saumya40-codes** [comment]
Thanks @vrunda2005 !

The main idea was to completely remove dependency on `contrib.rocks` as it takes time for it get updated.

You current commit make sure that contributors names are updated but it still currently depends on `contrib.rocks` for logo images, for this with name you can fetch an image url aswell and align them in the same way.


**something** like

```
CONTRIBUTORS_LOGOS="https://api.github.com/repos/${REPO_NAME}/contributors?per_page=10" | \
            jq -r '.[] | "- [@\(.login)](https://github.com/\(.login)) <img src=\"\(.avatar_url)\" alt=\"avatar\" style = \ "necessary styles required \"> "')

**Comment by vrunda2005** [comment]
Now I have changed this. I have tried to image in the circle but not working. 
Screenshot 2024-12-28 015035

File: .github/workflows/contributors_update.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+name: Update Contributors in README
+
+on:
+  push:
+    branches:
+      - main
+  schedule:
```
Comment (Saumya40-codes) [inline .github/workflows/contributors_update.yml]: we can remove this schedule part.

Contributors will change only during the push (if any new) so no need to run this workflow after every specific time period

**Comment by Saumya40-codes** [comment]
Look right 🙌

Thanks.

**Pull request merged**